spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+describe QueueItem do
+  it { should belong_to(:user) }
+  it { should belong_to(:video) }
+  
+  describe "#video_title" do
```

Comment:
You can use  ` it { is_expected.to delegate_method(:video_title).to(:video).as(:title) }` and then just use ` delegate :title, to: :video, prefix: :video`